src/audio/codec_adapter/codec/waves.c

```diff
@@ -724,7 +721,7 @@ waves_codec_process(struct processing_module *mod,
 	if (!codec->mpd.init_done)
 		waves_codec_init_process(dev);
 
-	memcpy_s(codes->mpd.in_buff, codec->mpd.in_buff_size,
+	memcpy_s(codec->mpd.in_buff, codec->mpd.in_buff_size,
```

Comment:
I see that you merged multiple fixes into a single commit. Ehh, I'll say this is fine.

Reply:
can we split this commit since this cant be tested by CI and should be easier to fault find when tested with the waves binary.